src/App.tsx

```diff
@@ -1,61 +1,110 @@
+import { useState } from 'react';
 import './App.scss';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleEntered, setTitleEntered] = useState(true);
```

Comment:
```suggestion
  const [isTitleEntered, setIsTitleEntered] = useState(true);
```
Same with other boolean state
![image](https://user-images.githubusercontent.com/96409563/215705639-c6d2d9ed-3276-4b15-b782-4c136700a8bc.png)